.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: Get Latest Version And Publish
```

Comment:
@yihui Do you need this workflow in the repo. I don't think it's necessary. What do you think?

Reply:
I removed it. :)